auth.py

```diff
@@ -17,8 +17,8 @@ def write_passwdb(pwdb):
 
 def pwhash(password):
     pwh = 0
-    for char in password:
-        pwh += ord(char)
+    for idx, char in enumerate(password):
+        pwh += (idx+1)*ord(char)
```

Comment:
Please put spaces around `+` operator.

Reply:
thank you very much for this suggestion, fixed this in f7a691bba74b91d579106d30cf02d63546c8729a